### ISSUE TITLE
dashboard no longer shows the version

### ISSUE THREAD

**Issue by A-Harby** [threefoldtecharchive/tfgrid_dashboard#455]
### Description

Version is not displayed and only display `no version provided`

### Logs/Alerts

![image](https://user-images.githubusercontent.com/109473873/212040848-24a935d3-e693-4663-add4-bf8e5e6f28f5.png)

**Comment by xmonader** [comment]
is that on devnet?

**Comment by A-Harby** [comment]
> is that on devnet?

Yess, devnet

**Comment by coesensbert** [comment]
I'm guessing it's because I removed the tag from the valuefile. 
https://docs.grid.tf/threefold/itenv_threefold_main/commit/74ae59904e4c2ca11548e34b1553bb4014148af1

I did this because we always have to change the version there and then push the version change to our ops repo. This is not needed for other services like graphql or substrate nodes. We checkout the correct version of the git repo and then deploy with the tag being empty in the valuefile.

Example of the processor: https://docs.grid.tf/threefold/itenv_threefold_main/src/branch/master/Kubernetes-clusters/hagrid-dev/applications/tfchain/graphql/processor/values.yaml#L11

This is probably just something we have to agree on how to handle versions with helm?
